### PR TITLE
feat: Central host UI files

### PR DESCRIPTION
**Pull request by FentPams**
## Description

The **branch** with preview files:

https://github.com/adobe/aem-experimentation/tree/gh-pages

**Todo:**

Configure the adobe.github.io to point to the gh-pages branch.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by FentPams** [comment]
@ramboz branch is ready :)

File: src/index.js [hunk 1]
```diff
@@ -918,6 +918,11 @@ export async function loadLazy(document, options = {}) {
     return;
   }
   // eslint-disable-next-line import/no-cycle
-  const preview = await import('./preview.js');
-  preview.default(document, pluginOptions);
+  const preview = await import('https://adobe.github.io/aem-experimentation/preview.js');
```
Comment (ramboz) [inline src/index.js:921]: Just realized this actually redirects to `https://opensource.adobe.com/aem-experimentation/preview.js`, so we could update the links to the final URL already :)

Comment (FentPams) [reply src/index.js]: Done. I also updated the path in gh-pages preview.js to `https://opensource.adobe.com/aem-experimentation/preview.css`

File: src/index.js [hunk 2]
```diff
@@ -918,6 +918,11 @@ export async function loadLazy(document, options = {}) {
     return;
   }
   // eslint-disable-next-line import/no-cycle
```
Comment (ramboz) [inline src/index.js:920]: Just need to update the eslint disable comment her so it doesn't fail the lining.
The "no-cycle" is not needed anymore, but you'll need a "no-unresolved"

**Review by ramboz — APPROVED** [review]

**Pull request merged**